specification/guestconfiguration/resource-manager/Microsoft.GuestConfiguration/preview/2018-06-30-preview/guestconfiguration.json

```diff
@@ -852,6 +1013,323 @@
         }
       },
       "description": "Reason and code for the compliance of the guest configuration assignment resource."
+    },
+    "SoftwareUpdateProfile": {
+        "description": "The profile of the computer in the context of software updates.",
```

Comment:
Please fix the indentations to indent 2 spaces all the way to line 1171.

Reply:
Fixed